client/components/common/tce-core/Discussion/ResolveButton.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="resolve-button">
```

Comment:
resolve-button-container ? Is this class used?

Reply:
Yes it is. It controlled outside within parent